target/linux/mediatek/files-5.10/drivers/leds/leds-ubnt-ledbar.c

```diff
@@ -67,34 +71,63 @@ static int ubnt_ledbar_apply_state(struct ubnt_ledbar *ledbar)
 	led_msg[UBNT_LEDBAR_TRANSACTION_BLUE_IDX] = ledbar->led_blue.brightness;
 	led_msg[UBNT_LEDBAR_TRANSACTION_GREEN_IDX] = ledbar->led_green.brightness;
 	led_msg[UBNT_LEDBAR_TRANSACTION_RED_IDX] = ledbar->led_red.brightness;
+	led_msg[UBNT_LEDBAR_TRANSACTION_LED_COUNT_IDX] = ledbar->led_count;
```

Comment:
This byte was zero before, with current core, the `led_count` would always be 1.

Reply:
No, the value at `led_msg[UBNT_LEDBAR_TRANSACTION_LED_COUNT_IDX]` was set to 0x01 in the definition of `led_msg` before this commit, defaulting `led_count` to 1 if the dtb entry is missing, just keeps it that way.